### PR TITLE
Cleanup work

### PR DESCRIPTION
**Pull request by alanyee**
Updated summary_op_util.py, head.py,  estimator.py, lookup_ops.py

### CODE REVIEW

File: tensorflow/python/ops/lookup_ops.py [hunk 1]
```diff
@@ -39,8 +39,7 @@
 from tensorflow.python.util.deprecation import deprecated
 
 
-# TODO(yleon): Remove this function.
-@deprecated("2017-03-02", "Use `tf.tables_initializer` instead.")
+@deprecated(None, "Use `tf.tables_initializer` instead.")
```
Comment (drpngx) [inline tensorflow/python/ops/lookup_ops.py]: @martinwicke I am not sure that this is a good idea to push the change date indefinitely. What do you think?

**Comment by drpngx** [comment]
Jenkins, test this please.

File: tensorflow/python/ops/lookup_ops.py (hunk 1 above)
Comment (alanyee) [reply tensorflow/python/ops/lookup_ops.py]: @drpngx He told me his opinion #12708

Comment (drpngx) [reply tensorflow/python/ops/lookup_ops.py]: Thanks @alanyee.

**Comment by drpngx** [comment]
GPU timeout.

Jenkins, test this please.

**Comment by alanyee** [comment]
@martinwicke @drpngx do these changes look good now?

**Review by martinwicke — APPROVED** [review]
Look good to me.

**Pull request merged**